Fotoblog.BLL/Services/TagService/ITagService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Fotoblog.BLL.Services.ServiceResultNS;
+using Fotoblog.Utils.ViewModels.Tags;
+
+namespace Fotoblog.BLL.Services.TagService
+{
+    public interface ITagService
```

Comment:
To jest TagService. Wynika z kontekstu że metody dotyczą tagów więc w nazwie słowa Tag być już nie musi. Pierdoła ale skraca kod.

Reply:
racja, fixed